src/components/Popup.component.tsx

```diff
@@ -24,11 +24,12 @@ export default function Popup({
     items,
     options: { highContrast },
     resetGame,
+    gameDate,
   } = useGlobal();
   const [copyButtonText, setCopyButtonText] = useState("Copy");
   console.log("highContrast", highContrast);
   const generateSummary = () => {
-    let summaryString = `Minecraftle ${new Date().toISOString().slice(0, 10)} ${
```

Comment:
Can't this be toLocaleString()?

Reply:
If you mean using the old code, just with `toLocaleString()`:

1. That will lead to the date formatting to be slightly different than the current, which I presume is not ideal.
2. The date to be used will still be from the timestamp of when the game has ended, not when the puzzle was opened.